### PR TITLE
Implementation of ModeledParameters table

### LINKED ISSUE #244: Make Modeled/Derived Parameter Table
Make a long table for derived and modeled parameters. first ingest can be the distances from Bailer Jones 2021 (#243 ).
Columns:
- Source
- Parameter - constrained name space.
- Value
- Low value
- High value
- maximum likelihood 
- mean marginalized posterior
- variance of marginalized posterior
- Comment - could include what "value", "low", and "high" correspond to.
- Reference

### PR DESCRIPTION
**Pull request by dr-rodriguez**
Initial implementation of ModeledParameters table.
This is fairly simple and includes the enumeration constraints for that table. I have encountered some issues with enumerations so I'm trying to keep them as simple as possible (see #271)

Do we want to create a test that loads up some fake ModeledParameters data?

Closes #244

### CODE REVIEW

File: simple/schema.py [hunk 1]
```diff
@@ -97,6 +98,16 @@ class Gravity(enum.Enum):
...
 
 
+class Parameter(enum.Enum):
+    """Enumeration for derived/inferred parameters"""
+    mass = 'mass'
+    radius = 'radius'
+    log_g = 'log_g'
+    T_eff = 'T_eff'
+    metallicity = 'metallicity'  # Z
+    CO = 'CO'  # C/O
```
Comment (kelle) [inline simple/schema.py:108]: maybe we should be more pedantic here. I think `CO` is too ambiguous/misleading. Maybe `C/O ratio`? I dont think there's a problem with the slash in the string is there? If so, `C-to-O ratio`?

Comment (dr-rodriguez) [reply simple/schema.py:108]: I need to confirm (and maybe write a test for) but I'm 80% sure that what gets stored in the database is the variable, not the string. Which means things like `C/O` and `C-to-O` are invalid.

Comment (kelle) [reply simple/schema.py:108]: C_to_O_ratio

**Comment by kelle** [comment]
Let's not use an enumeration for Parameter's, let's use a reference table.

File: simple/schema.py (hunk 1 above)
Comment (dr-rodriguez) [reply simple/schema.py]: Switching to an actual table, so the enumeration question is no longer relevant

**Comment by dr-rodriguez** [comment]
Explicitly populate the Parameters table with: mass, radius, log_g, T_eff, metallicity, C-to-O ratio.

**Comment by dr-rodriguez** [comment]
Should Parameters have units? If so, what should we store for these? (I imagine that some are NULL, which is fine)

**Review by kelle — CHANGES_REQUESTED** [review]

**Review by kelle — APPROVED** [review]

**Pull request merged**